daemon/src/model.rs

```diff
@@ -83,3 +84,21 @@ impl Display for TakerId {
         self.0.fmt(f)
     }
 }
+
+#[allow(dead_code)]
+#[derive(Debug, thiserror::Error, Clone, Copy)]
+pub enum WalletError {
+    #[error("Wallet sync failure")]
+    Sync,
+    #[error("Wallet balance retrieval failure")]
+    Balance,
+    #[error("Wallet address retrieval failure")]
+    Address,
+}
+
+#[derive(Debug, Clone)]
+pub enum WalletInfo {
+    Syncing,
```

Comment:
Does it bring much benefit to tell the user that we are syncing? What if we output a timestamp next to the wallet with "Last updated at"?

Reply:
I think it does somewhat help, but it is actually only used initially when we set it to be a `Spinner`. But given that the `Spinner` is actually not based on state but just an initial value we might as well simplify this.

Reply:
Our model of updates on the feed works best if you don't depend on the updates being sent in any particular order. Each update should map to a clear UI state.

We can do a "syncing" update but I find it a bit weird because the UI doesn't actually know about the state.
I would probably just send out the final update and add an animation everytime it changes, plus update the timestamp so the user knows it is up2date.

Reply:
One problem I see with that approach is, that if the wallet takes significant time for syncing the user just won't know and might assume that the balance is up to date (even though it might be outdated).
I remember we had problems with that in Ambrosia at some point - not sure that scenario applies here though. 
I can go with the timestamp, but I don't know if that helps. The timestamp does not tell the user anything because there is no periodic balance change. So, the timestamp is no indicator for the balance being up to date or not. It just shows when the last change happened. I don't see that as something very useful.

Reply:
If we remove the optimisation with sending out updates, the timestamp will refresh with your interval and hence accurately reflect, how often you are actually checking the balance internally.